pom.xml

```diff
@@ -280,7 +284,13 @@
         <xml-matchers.version>0.10</xml-matchers.version>
         <zookeeper.version>3.5.7</zookeeper.version>
     </properties>
-
+    <repositories>
+        <repository>
+            <id>jcenter</id>
```

Comment:
what is this for?

Reply:
This is for pty4j. Interestingly, [Pty4j was not part of maven central until Nov 2021 ](https://mvnrepository.com/artifact/org.jetbrains.pty4j/pty4j)(when this was introduced). I can now see [Pty4j in maven central ](https://mvnrepository.com/artifact/org.jetbrains.pty4j/pty4j) 

I'll use the new version of Pty4j from maven central and see if it works.
Thank you soo much for catching this!

Reply:
unfortunately this is needed for [purejavacomm](https://mvnrepository.com/artifact/com.github.purejavacomm/purejavacomm) which is only in jcenter.

Reply:
Unfortunately, this is needed for [PurejavaComm library](https://mvnrepository.com/artifact/org.jetbrains.pty4j/purejavacomm?repo=jcenter)  that is only available in jcenter.